advisories/hackage/bzlib/HSEC-2024-0002.md

```diff
@@ -9,27 +9,35 @@ aliases = ["CVE-2019-12900"]
 type = "DISCUSSION"
 url = "https://gnu.wildebeest.org/blog/mjw/2019/08/02/bzip2-and-the-cve-that-wasnt/"
 
+[[references]]
+type = "DISCUSSION"
+url = "http://scary.beasts.org/security/CESA-2008-005.html"
+
+[[references]]
+type = "DISCUSSION"
+url = "https://access.redhat.com/security/cve/cve-2019-12900"
+
 [[references]]
 type = "FIX"
 url = "https://sourceware.org/git/?p=bzip2.git;a=commit;h=7ed62bfb46e87a9e878712603469440e6882b184"
 
 [[affected]]
 package = "bzlib"
-cvss = "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
+cvss = "CVSS:3.0/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L"
```

Comment:
I propose instead: `/C:N/I:L/A:N`.  But I might be missing or misunderstanding something, so let us discuss.

Reply:
That sounds good to me, the new score is Medium, 4.0